lib/elixir/lib/task/supervised.ex

```diff
@@ -150,6 +152,7 @@ defmodule Task.Supervised do
           reason: reason
         }
       }) do
+    # TODO confirm if we need to add process_label
```

Comment:
I added [a9b727e](https://github.com/elixir-lang/elixir/pull/13406/commits/a9b727e93395945963d0f5810adb145477f977d9) but I am unsure how to test it - it currently isn't.

Sorry for the noob question, in which case do we use `report_cb` instead of the logger's formatting?

Reply:
If you remove our translator, it should fallback to report_cb, I believe. Otherwise, if you remove our whole handler and rely on Erlang built-in logger handler.

Reply:
@josevalim I think I got it [20346fb](https://github.com/elixir-lang/elixir/pull/13406/commits/20346fb5d204ca40bda45fc3d5276bb5510a0c02)